frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/networking-review.tsx

```diff
@@ -1,34 +1,46 @@
 import * as React from 'react';
-import * as _ from 'lodash';
 import { connect } from 'react-redux';
 import { VMWizardNetwork } from '../../types';
 import { getNetworks } from '../../selectors/selectors';
-import { ReviewList } from './review-list';
 import { NetworkInterfaceWrapper } from '../../../../k8s/wrapper/vm/network-interface-wrapper';
 import { NetworkWrapper } from '../../../../k8s/wrapper/vm/network-wrapper';
+import { Table, TableBody, TableHeader, TableVariant } from '@patternfly/react-table';
```

Comment:
can you add the modifier to the table `pf-m-grid-2xl` it will add a sooner breakpoint.

Reply:
Done!